src/progress/pct/compile.p

```diff
@@ -300,8 +302,8 @@ PROCEDURE compileXref.
     ASSIGN opComp = 5.
   END.
   ELSE DO:
-    /* Does .r file exists ? */
-    ASSIGN RCodeTS = getTimeStampDF(OutputDir, ipOutFile).
+    /* Does .r file exists ?, if DestDir = unknown rcode will be located in the same directory as the source : ipInDir */
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Line is 121 characters long, exceeding maximum value of 120 [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=rssw-oe-main%3Aeu.rssw.antlr.proparse.checks.LinesTooLong)